neo/public/javascripts/common/services.js

```diff
@@ -226,13 +226,14 @@ angular.module('cloudberry.common', [])
             asterixService.totalCount = result.value[0][0].count;
             break;
           case "error":
-            console.error(result.toString());
+            console.error(result);
             asterixService.errorMessage = result.value;
             break;
           case "done":
             break;
           default:
-            console.error("ws get unknown data: " + result.toString());
+            console.error("ws get unknown data: ");
+            console.error(result);
```

Comment:
JS is even simpler than scala 
`console.error("ws get unknown data: ", result);`